warehouse/manage/views.py

```diff
@@ -1595,6 +1627,26 @@ def delete_project_role(project, request):
                     "target_user": role.user.username,
                 },
             )
+
+            owner_roles = (
+                request.db.query(Role)
+                .filter(Role.project == project)
+                .filter(Role.role_name == "Owner")
+                .all()
+            )
+            owner_users = {owner.user for owner in owner_roles}
+            send_collaborator_removed_email(
+                request,
+                owner_users,
```

Comment:
```suggestion
                collaborators,
```

Reply:
I also think we want to do `collaborators.discard(role.user)` before we sent this as well.